0005-propose-new-adrs-as-pull-requests.md

```diff
@@ -0,0 +1,26 @@
+# 5. Propose new ADRs as pull requests
```

Comment:
Can we clarify whether 1 ADR = 1 PR or if it's more flexible? Obviously this first one is a different case

Reply:
Makes sense